contracts/tokenbridge/arbitrum/L2AtomicTokenBridgeFactory.sol

```diff
@@ -49,8 +56,43 @@ contract L2AtomicTokenBridgeFactory {
             );
         }
 
-        // transfer ownership to rollup's owner
-        ProxyAdmin(proxyAdmin).transferOwnership(rollupOwner);
+        // deploy multicall
+        Create2.deploy(0, _getL2Salt(OrbitSalts.L2_MULTICALL), _creationCodeFor(l2Code.multicall));
```

Comment:
Should we use a proxy in case the multicall implementation changes? Or has it been stable for a long time?

Reply:
hmm good q, not sure if it is worth it. Arb multicall hasn't been changed last 2 year as far as I can see. @gzeoneth what do you think?